### PR TITLE
fix(@desktop/chat): adjust add members default picture

### LINKED ISSUE #6971: The default profile image is not shown in the group chat members' list
# Bug Report

## Steps to reproduce
1. Add a group chat with users without profile image
2. In the secondary navigation tab right click on the group chat - View Members - Add members
3. Check the user's default image

#### Expected behavior
The default profile image is shown correctly in the group chat members' list
#### Actual behavior
The default profile image is not shown in the group chat members' list
image

### Additional Information

- Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/master/59/
- Operating System: macOS Monterey

### PR DESCRIPTION
**Pull request by faust4exe**
Fixes #6971

### What does the PR do

Add icon settings to ContactListPanel to cover no image case.

### Affected areas

Desktop Chat

### Screenshot of functionality 

https://user-images.githubusercontent.com/6445843/185596070-d25c4daf-f6ab-4c8a-93db-b2fa817dc439.mp4

### CODE REVIEW

**Comment by faust4exe** [comment]
@MishkaRogachev Can you please test this small fix and confirm it works as expected ?

File: ui/app/AppLayouts/Chat/panels/ContactListPanel.qml [hunk 1]
```diff
@@ -34,7 +35,14 @@ StatusListView {
 
         title: !model.displayName.endsWith(".eth") && !!model.localNickname ?
                     model.localNickname : Utils.removeStatusEns(model.displayName)
-        image.source: Global.getProfileImage(model.pubKey)
+        image.source: model.icon
+        icon: StatusIconSettings {
+            color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.pubKey)]
```
Comment (MishkaRogachev) [inline ui/app/AppLayouts/Chat/panels/ContactListPanel.qml:40]: It's better to check array borders here with some default color

**Review by MishkaRogachev — CHANGES_REQUESTED** [review]

File: ui/app/AppLayouts/Chat/panels/ContactListPanel.qml (hunk 1 above)
Comment (faust4exe) [reply ui/app/AppLayouts/Chat/panels/ContactListPanel.qml:40]: Can you show how you expect it to look like ? Also we are missing that kind of check on many other places.

Comment (MishkaRogachev) [reply ui/app/AppLayouts/Chat/panels/ContactListPanel.qml:40]:
```
color: {
    const pubKeyColorId = Utils.colorIdForPubkey(model.pubKey);
    if (pubKeyColorId < 0 || pubKeyColorId >=  Theme.palette.userCustomizationColors) {
       // emit some warning
        return someDefaultColor;
   }
   return Theme.palette.userCustomizationColors[pubKeyColorId]
}
```
Maybe it's not necessary but if some one change Theme.palette this check helps to discover problem

**Review by MishkaRogachev — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

**Pull request merged**